bundles/org.openhab.binding.frenchelectricitytariffs/README.md

```diff
@@ -0,0 +1,72 @@
+# French Electricity Tariffs Binding
+
+This binding provides regulated electricity prices in France.
+
+This can be used to plan energy consumption, for example to calculate the cheapest period for running a dishwasher or charging an EV.
+
+## Supported Things
+
+The binding offers things for the two usual tariff classes (proposed by example by EDF).
+
+- `base`: This is the basic subscription with a fixed kWh price.
+- `hphc`: Alternative subscription offering variable price in a given hour set (low hours/high hours).
+
+
+## Thing Configuration
+
+Things (both `base` and `hphc`) only offers the configuration of the power output of the electrical delivery point (Linky terminal).
+
+| Name                  | Type    | Description                                 | Default       | Required |
+|-----------------------|---------|---------------------------------------------|---------------|----------|
+| puissance             | integer | PDL power output (in kVA)                   | 6             | no       |
+
+
+## Channels
+
+### `base` Tariff Thing
+
+All channels are read-only.
+
+| Channel      | Type               | Description                             | Advanced |
+|--------------|--------------------|-----------------------------------------|----------|
+| fixed-ttc    | Number:Currency    | Yearly fixed price including taxes      | No       |
+| variable-ttc | Number:EnergyPrice | Energy price in €/kWh including taxes   | No       |
+| tariff-start | DateTime           | Beginning date for this tariff          | Yes      |
+| fixed-ht     | Number:Currency    | Yearly fixed price excluding taxes      | Yes      |
+| variable-ht  | Number:EnergyPrice | Energy price in €/kWh excluding taxes   | Yes      |
```

Comment:
What is meant with "taxes" here, is that all kinds of taxes - also VAT? Maybe this could be used for inspiration, depending on what is exactly included/excluded: https://www.openhab.org/addons/bindings/energidataservice/#value-added-tax